euclidean_gcd/euclidean_gcd.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import "fmt"
+
+// EuclideanGCDIterative finds GCD of given numbers by iterative method
+func EuclideanGCDIterative(a int, b int) int {
+	for b != 0 {
+		tmp := b
+		b = a % b
+		a = tmp
+	}
+	return a
+}
+
+// EuclideanGCDRecursive finds GCD of given numbers by recursion
```

Comment:
Please show the Recurrence equation of the algorithm, and also give the time complexity as well